### PR TITLE
Further docs work

### PR DESCRIPTION
**Pull request by danni**
Use aloe_sphinx, split docs into multiple pages, update README, correctly link to Aloe docs.

### CODE REVIEW

**Comment by danni** [comment]
This is failing because `aloe_sphinx` is not in a released version of Aloe yet.

File: README.md [hunk 1]
```diff
@@ -1,9 +1,41 @@
 Aloe-Django
 -------------
 
-Utilities for using Aloe with Django:
+Utilities for using [Aloe](https://github.com/koterpillar/aloe) with
+Django(http://djangoproject.com):
```
Comment (koterpillar) [inline README.md]: []

File: README.md [hunk 2]
```diff
@@ -1,9 +1,41 @@
...
 Aloe and Aloe-Django are based on [Lettuce](http://lettuce.it/).
+
+License
+=======
+
+Aloe-Django - Django utils for Aloe based on Lettuce
+
+Copyright (C) <2015> Alexey Kotlyarov <a@koterpillar.com>
+
+Copyright (C) <2015 Danielle Madeley <danielle@madeley.id.au>
```
Comment (koterpillar) [inline README.md]: `>`

File: tools/check_installs [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+deactivate || true
+rm -rf check_install_venv
+virtualenv check_install_venv
+. check_install_venv/bin/activate
+
+# Install the package
+pip install check_install_dist/*.tar.gz
+
+# Smoke test
+aloe --version
```
Comment (koterpillar) [inline tools/check_installs]: Is this right?

**Pull request merged**